source/_docs/wordpress-custom-code.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Managing Custom Code for WordPress
+description: Learn how to manage custom plugins or themes for WordPress sites using GitHub Updater.
+categories: [wordpress]
+tags: [terminus, code]
+---
+Extending WordPress with custom code is a common part of the development lifecycle. In order to optimize workflows, it's essential that extensions are managed independently to facilitate reusability. For changes to persist when swapping themes, there must be a separation between function and design. This means no more dropping code into the current theme's `functions.php` file and no more pasting snippets you want to reuse in other projects.
+
+Instead, manage custom code separate from your projects within site-specific [plugins](https://codex.wordpress.org/Writing_a_Plugin). Then scope out an update strategy to easily distribute changes, such as [GitHub Updater](https://github.com/afragen/github-updater).
+
+<div class="alert alert-info">
+<h4>Note</h4>
+Pantheon does not support Git submodules (placing a Git repository within a subdirectory of your site’s repository).
+</div>
+## Before You Begin
+
+1. Separate custom code into site-specific plugins or themes stored within individual repositories outside of your project.
+2. Host your custom extensions remotely on [GitHub](https://github.com/), [Bitbucket](https://bitbucket.org), or [GitLab](https://about.gitlab.com/).
+
+For more details on separating custom code, see related [blog post](https://pantheon.io/blog/wordpress-development-git).
+
+## GitHub Updater
+The [GitHub Updater](https://github.com/afragen/github-updater) plugin extends the existing notification and update mechanisms within WordPress to plugins/themes hosted outside of the official WordPress repository on GitHub, Bitbucket, or GitLab.
+
+1. Add support for GitHub Updater within your plugins and themes by providing the required declarations. The following [plugin example](https://github.com/afragen/github-updater/#plugins) would be placed within the plugin's header:
+
+ ```
```

Comment:
May want to not use the backticks and wrap this section in `<pre>...<b>...</b></pre>` tags so you can bold that actual header lines required.

<pre>
/*
Plugin Name:       GitHub Updater
Plugin URI:        https://github.com/afragen/github-updater
Description:       A plugin to automatically update GitHub, Bitbucket or GitLab hosted plugins and themes. It also allows for remote installation of plugins or themes into WordPress.
Version:           1.0.0
Author:            Andy Fragen
License:           GNU General Public License v2
License URI:       http://www.gnu.org/licenses/gpl-2.0.html
Domain Path:       /languages
Text Domain:       github-updater
<b>GitHub Plugin URI: https://github.com/afragen/github-updater
GitHub Branch:     master</b>
*/
</pre>